### DISCUSSION TITLE
CEF improvements

### DISCUSSION THREAD

**Discussion by CrosRoad95** [multitheftauto/mtasa-blue#284] [Ideas]
Mainly issue about CEF is bridge between lua <- -> js, it sucks.

There should be added functions that can simplify communication between CEF and lua
1. Execute functions by name, like `call` function.
2. Getting values under variable.
3. Setting values.

**Comment by jushar** [comment]
Which direction do you mean particularly?
Lua to JavaScript or vice versa?

**Comment by CrosRoad95** [comment]
both, from javascript to lua, and lua to javascript
but javascript to lua could be whitelisted
